### PR TITLE
LPS-154750 Improvements for live-edited while Page editing

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Layout.scss [hunk 1]
```diff
@@ -193,6 +194,13 @@ html#{$cadmin-selector} {
 
 			&--active {
 				box-shadow: inset 0 0 0 $editablesBorderWidth $cadmin-primary;
+
+				@each $type in $editableTypes {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Layout.scss]: This is awesome :heart_eyes:

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js [hunk 2]
```diff
@@ -322,7 +341,21 @@ function FragmentContentInteractionsFilter({
 		props.onMouseOverCapture = hoverEditable;
 	}
 
-	return <div {...props}>{children}</div>;
+	return (
+		<>
+			<div {...props}>{children}</div>
+			{triggerElement ? (
+				<Tooltip
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js:351]: Not sure if it is the right place for the tooltip. It is a bit weird that we are only sometimes rendering UI

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js [hunk 3]
```diff
@@ -322,7 +341,21 @@ function FragmentContentInteractionsFilter({
...
+					alignPosition="bottom"
+					customPosition={tooltipPosition}
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js:353]: I think we should handle this behavior inside Tooltip component. It is a bit weird that you have to pass a `positionElement`, but then if you specify `customPosition` too, `positionElement` is ignored.

Maybe we can more specific here using props, something like:

```
verticalAlign: 'top' | 'bottom' | 'cursor' | element
horizontalAlign: 'left' | 'right' | 'cursor' | element
```

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/Tooltip.js [hunk 4]
```diff
@@ -34,22 +45,37 @@ export function Tooltip({
...
+			showTimeoutId = setTimeout(() => {
+				setTooltipStyle(position);
+			}, delay);
 		};
 
-		const handleMouseOver = () => {
+		if (customPosition) {
+			const rect = positionElement.getBoundingClientRect();
+
+			setTooltipStypeTimeout({
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/Tooltip.js:59]: Typo here :3

**Review by p2kmgcl — COMMENTED** [review]
I just added some comments, but we will discuss this later on slack :partying_face:

**Comment by p2kmgcl** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js [hunk 5]
```diff
@@ -237,7 +237,7 @@ function FragmentContentInteractionsFilter({
 				) {
 					requestAnimationFrame(() => {
 						activeEditable.element.addEventListener(
-							'dblclick',
+							'click',
```
[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js:240]: Hello darkness my old friend

[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentInteractionsFilter.js:240]: I think we have to at least hide this in a FF

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**